libs/langgraph/langgraph/types.py

```diff
@@ -133,6 +133,11 @@ class Interrupt:
     when: Literal["during"] = "during"
 
 
+class BulkUpdate(NamedTuple):
```

Comment:
nit - it's a single update so should probably just be called `Update` or `StateUpdate`. also, should this just be a typeddict? (don't have a super strong preference)

Reply:
Renamed to `StateUpdate`, leaving TypedDict vs NamedTuple up for grabs.